lib/core/rlocks.nim

```diff
@@ -48,7 +48,7 @@ proc release*(lock: var RLock) {.inline.} =
   ## Releases the given lock.
   releaseSys(lock)
 
-template withRLock*(lock: var RLock, code: untyped): untyped =
+template withRLock*(lock: RLock, code: untyped) =
```

Comment:
Why remove the `var`?

Reply:
No need for it in a template.

Reply:
But it is needed when you actually wanna use the `template`, right (because `acquire` requires `var`)? So why not keep it here, despite it not being strictly necessary, for better documentation?

Reply:
I don't agree with you, I think its needless duplication, in fact the manual only mentions typed and untyped for templates.